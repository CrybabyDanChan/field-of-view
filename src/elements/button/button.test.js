import React from 'react';
import {ReactDOM} from 'react';
import renderer from 'react-test-renderer';
import Button from './index';

it('Correctly renderer', () => {
  // Arrange
  const component = renderer.create(<Button>Hello</Button>);

  // Act
  const tree = component.toJSON();

  // Assert
  expect(tree).toMatchSnapshot();
});
