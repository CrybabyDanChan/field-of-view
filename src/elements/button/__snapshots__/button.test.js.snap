// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly renderer 1`] = `
<div
  className="buttonstyle__ButtonWrapper-sc-2v3gkn-0 koDNiB"
>
  <button
    className="button"
    type="button"
  >
    Hello
  </button>
</div>
`;
